docs/help.md

```diff
@@ -0,0 +1,27 @@
+### Commands
+`bot adapter` - Reply with the adapter
```

Comment:
What is the adapter?

Reply:
It is just small `hubot` feature that returns adapter. In our case it is gitter adapter)